Makefile

```diff
@@ -65,18 +69,67 @@ clean-tools:
 clean-all: clean clean-tools
 components: $(BIN)
 	$(BIN) components
-install-tools: install-go-junit-report $(TOOLS_BIN_NAMES)
+install-tools: $(TOOLS_BIN_NAMES) install-goreleaser-pro install-go-junit-report
 snapshot: .goreleaser.yaml $(GORELEASER)
 	$(GORELEASER) release --snapshot --clean --parallelism 2 --skip archive,sbom --fail-fast
-release: .goreleaser.yaml $(GORELEASER)
-	$(GORELEASER) release --clean --parallelism 1 --release-notes $(RELEASE_NOTES)
 
 $(TOOLS_BIN_DIR):
 	mkdir -p $@
 
 $(TOOLS_BIN_NAMES): $(TOOLS_MOD_DIR)/go.mod | $(TOOLS_BIN_DIR)
 	cd $(TOOLS_MOD_DIR) && go build -o $@ -trimpath $(filter %/$(notdir $@),$(TOOLS_PKG_NAMES))
 
+OS := $(shell uname)
+ARCH := $(shell uname -m)
+
+ifeq ($(ARCH), 'amd64')
+	ARCH=x86_64
+else ifeq ($(ARCH), 'aarch64')
+	ARCH=arm64
+endif
+
+EXT := tar.gz
+ifeq ($(OS), 'windows')
+	EXT='zip'
+endif
+
+# Construct binary name and URL
+BINARY_NAME := goreleaser-pro_$(OS)_$(ARCH).$(EXT)
```

Comment:
Nit: I think `ARCHIVE_NAME` would be a better name here since we're working with a tar/zip archive instead of the binary itself.

Reply:
good point, done